modules/virt-configuring-ssh.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-ssh.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-ssh_{context}"]
+= Configuring SSH for quick virtual machine access
+
+You can configure your local SSH client to quickly connect to your virtual machine (VM). This configuration makes it possible to leverage `virtctl` to relay an SSH command to the target VM.
+

```

Comment:
The hint 
```
This method is useful for troubleshooting a VM because it requires little time to set up. The connection is slow, but it works consistently even if you are behind a firewall or if the cluster does not have a load balancer.

```

from `virtctl ssh` applies also here, because it is technically the same. It might help to avoid duplication, to delay this PR until https://github.com/openshift/openshift-docs/pull/61419 is merged.